README.md

```diff
@@ -53,4 +53,4 @@ You can then run the development server:
 ```
 And navigate to `http://127.0.0.1:8000/`.
 
-From here you can make Register an account and start adding your plants!
\ No newline at end of file
+From here you can make Register an account and start adding your plants!
```

Comment:
What has actually changed on this line?

Reply:
Nothing as far as I know - I didn't edit this line. But it looks like white-space.